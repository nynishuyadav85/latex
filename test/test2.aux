\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces these value $ f(x) $}}{4}{table.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {1}Lists}{7}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}learning}{7}{subsection.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2}thinking}{7}{subsection.1.2}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}formatting}{7}{section.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}text}{7}{subsection.2.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}words}{7}{subsection.2.2}\protected@file@percent }
\gdef \@abspage@last{7}
