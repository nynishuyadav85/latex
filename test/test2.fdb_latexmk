# Fdb version 4
["pdflatex"] 1741096876.11393 "/Users/nishantmac/Documents/latex/test/test2.tex" "test2.pdf" "test2" 1741096876.51905 0
  "/Users/nishantmac/Documents/latex/test/test2.tex" 1741096875.90978 207 16ab24dd1cb4809a88139be7a16e5a4c ""
  "/usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi10.pfb" 1248133631 36299 5f9df58c2139e7edcf37c8fca4bd384d ""
  "/usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmr10.pfb" 1248133631 35752 024fb6c41858982481f6968b5fc26508 ""
  "/usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmr7.pfb" 1248133631 32762 224316ccc9ad3ca0423a14971cfa7fc1 ""
  "/usr/local/texlive/2024/texmf-dist/tex/latex/base/article.cls" 1705352648 20144 147463a6a579f4597269ef9565205cfe ""
  "/usr/local/texlive/2024/texmf-dist/tex/latex/base/size10.clo" 1705352648 8448 dbc0dbf4156c0bb9ba01a1c685d3bad0 ""
  "/usr/local/texlive/2024/texmf-dist/tex/latex/l3backend/l3backend-pdftex.def" 1708463273 30006 3d512c0edd558928ddea1690180ef77e ""
  "/usr/local/texlive/2024/texmf-dist/web2c/texmf.cnf" 1708706663 41649 5d6ae549fbbcb850a863f69aa41f3d10 ""
  "/usr/local/texlive/2024/texmf-var/fonts/map/pdftex/updmap/pdftex.map" 1741071558.9985 5467645 128b85b7cde5f5edc5cb7f1dd7ff8736 ""
  "/usr/local/texlive/2024/texmf-var/web2c/pdftex/pdflatex.fmt" 1741071509 8221444 a65402a1fcc633c1b3dbf910d1469ed4 ""
  "/usr/local/texlive/2024/texmf.cnf" 1710266656 577 e590dabc9e28c5b61546e15f63eebcdf ""
  "test2.aux" 1741096876.48994 32 3985256e7290058c681f74d7a3565a19 "pdflatex"
  "test2.tex" 1741096875.90978 207 16ab24dd1cb4809a88139be7a16e5a4c ""
  (generated)
  "test2.aux"
  "test2.log"
  "test2.pdf"
  (rewritten before read)
