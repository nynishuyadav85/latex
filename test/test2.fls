PWD /Users/nishantmac/Documents/latex/test
INPUT /usr/local/texlive/2024/texmf.cnf
INPUT /usr/local/texlive/2024/texmf-dist/web2c/texmf.cnf
INPUT /usr/local/texlive/2024/texmf-var/web2c/pdftex/pdflatex.fmt
INPUT /Users/nishantmac/Documents/latex/test/test2.tex
OUTPUT test2.log
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/base/article.cls
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/base/article.cls
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/base/size10.clo
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/base/size10.clo
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/base/size10.clo
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amsmath.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amsmath.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amsopn.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amstext.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amstext.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amsgen.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amsgen.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amsbsy.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amsbsy.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/amsmath/amsopn.sty
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/l3backend/l3backend-pdftex.def
INPUT /usr/local/texlive/2024/texmf-dist/tex/latex/l3backend/l3backend-pdftex.def
INPUT ./test2.aux
INPUT ./test2.aux
INPUT test2.aux
OUTPUT test2.aux
INPUT /usr/local/texlive/2024/texmf-dist/fonts/map/fontname/texfonts.map
INPUT /usr/local/texlive/2024/texmf-dist/fonts/tfm/public/amsfonts/cmextra/cmex7.tfm
INPUT /usr/local/texlive/2024/texmf-dist/fonts/tfm/public/amsfonts/cmextra/cmex7.tfm
INPUT /usr/local/texlive/2024/texmf-dist/fonts/tfm/public/cm/cmbx10.tfm
OUTPUT test2.pdf
INPUT /usr/local/texlive/2024/texmf-var/fonts/map/pdftex/updmap/pdftex.map
INPUT test2.aux
INPUT /usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx10.pfb
INPUT /usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmex10.pfb
INPUT /usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi10.pfb
INPUT /usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi7.pfb
INPUT /usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmr10.pfb
INPUT /usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmr5.pfb
INPUT /usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmr7.pfb
INPUT /usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmsy10.pfb
INPUT /usr/local/texlive/2024/texmf-dist/fonts/type1/public/amsfonts/cm/cmsy7.pfb
