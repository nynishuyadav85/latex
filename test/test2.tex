\documentclass[]{article}

\begin{document}

Hello, This my first \LaTeX\ document!

The rectangle is of length $(x+2)$ and $(x+3)$.
The Equation $${A(x)=x^2+4x+3}$$ gives the area of rectangle

-----------------------------------------------------------------------------------------

\textbf{Common mathmatical Notation}

SuperScript $$2x^3$$
$$2x^{34}$$
$$2x^{2x+4}$$
$$2x^{3x^{54}}$$

SubScripts  $${x_1}$$
$${x_{12}}$$
$${x_{1_{2_{3_4}}}}$$
$${a_1}, {a_2}, \ldots {a_{100}}$$

Greek Letters   

$$\pi$$
$$\Pi$$
$$\alpha$$
$$\aleph$$
$$A= \pi r^2$$

\end{document}