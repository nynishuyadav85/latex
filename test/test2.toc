\contentsline {section}{\numberline {1}Lists}{7}{section.1}%
\contentsline {subsection}{\numberline {1.1}learning}{7}{subsection.1.1}%
\contentsline {subsection}{\numberline {1.2}thinking}{7}{subsection.1.2}%
\contentsline {section}{\numberline {2}formatting}{7}{section.2}%
\contentsline {subsection}{\numberline {2.1}text}{7}{subsection.2.1}%
\contentsline {subsection}{\numberline {2.2}words}{7}{subsection.2.2}%
