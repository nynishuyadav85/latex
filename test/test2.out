\BOOKMARK [1][-]{section.1}{\376\377\000L\000i\000s\000t\000s}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{\376\377\000l\000e\000a\000r\000n\000i\000n\000g}{section.1}% 2
\BOOKMARK [2][-]{subsection.1.2}{\376\377\000t\000h\000i\000n\000k\000i\000n\000g}{section.1}% 3
\BOOKMARK [1][-]{section.2}{\376\377\000f\000o\000r\000m\000a\000t\000t\000i\000n\000g}{}% 4
\BOOKMARK [2][-]{subsection.2.1}{\376\377\000t\000e\000x\000t}{section.2}% 5
\BOOKMARK [2][-]{subsection.2.2}{\376\377\000w\000o\000r\000d\000s}{section.2}% 6
